docs/source/index.rst

```diff
@@ -4,6 +4,14 @@ Hyperledger Iroha documentation
 
 .. image:: ../image_assets/iroha_logo.png
 
+.. warning::
+    Please note that support for Hyperledger Iroha v1 is limited because it is no longer actively developed.
+
+    The core team focuses on `Hyperledger Iroha v2 <https://github.com/hyperledger/iroha/tree/iroha2-dev#hyperledger-iroha>`_, a complete rewrite of Iroha in Rust.
+
+    These versions are incompatible. We recommend using Iroha 2 instead of Iroha 1 for new projects.
+    You can read about the differences in the `Iroha 2 documentation <https://hyperledger.github.io/iroha-2-docs/guide/iroha-2.html>`_.
```

Comment:
```suggestion
    Please note that Hyperledger Iroha v1 is **no longer under active development** and the support for Iroha v1 is limited. The core development team focuses on `Hyperledger Iroha v2 <https://github.com/hyperledger/iroha/tree/iroha2-dev#hyperledger-iroha>`_, a complete rewrite of Iroha in Rust.
    
    For new projects, **we recommend using Iroha v2 instead of Iroha v1**. If you are using Iroha v1, we encourage you to plan an upgrade and switch to using Iroha v2.
    
    The long-term supported version of Iroha v2 (`iroha2-lts`) is your best choice if you want to use Iroha in production and get tech support from our side. Unfortunately, we cannot offer the same support for Iroha v1, so please consider using Iroha v2.

    Note that Iroha v1 and Iroha v2 are incompatible. You can read about their differences in the `Iroha 2 documentation <https://hyperledger.github.io/iroha-2-docs/guide/iroha-2.html>`_.
```

Reply:
Thanks!